File: src/SOP8/Base/Account/Base.Account.IBLL/Models/UserOption.cs
using System.Collections.Generic;

namespace Base.Account.IBLL.Models
{
    public class UserOption
    {
        private string m_strCategory = null;
        private string m_strSubCategory = null;
        private List<string> m_strValues = new List<string>();

        public string Category
        {
            get { return m_strCategory; }
            set { m_strCategory = value; }
        }

        public string SubCategory
        {
            get { return m_strSubCategory; }
            set { m_strSubCategory = value; }
        }

        public List<string> Values
        {
            get { return m_strValues; }
            set { m_strValues = value; }
        }
    }
}

File: src/SOP8/Egovframework/Base/base-account-ibll/src/main/java/egovframework/base/account/ibll/models/UserOption.java
package egovframework.base.account.ibll.models;

import java.util.ArrayList;
import java.util.List;

public class UserOption {

    private String category = null;
    private String subCategory = null;
    private List<String> values = new ArrayList<>();

    public String getCategory() {
        return category;
    }

    public void setCategory(String category) {
        this.category = category;
    }

    public String getSubCategory() {
        return subCategory;
    }

    public void setSubCategory(String subCategory) {
        this.subCategory = subCategory;
    }

    public List<String> getValues() {
        return values;
    }

    public void setValues(List<String> values) {
        this.values = values;
    }
}